transbank/onepay/transaction.py

```diff
@@ -118,3 +148,28 @@ def create(cls, shopping_cart: ShoppingCart, channel = None, external_unique_num
             raise TransactionCreateError("The response signature is not valid.", -1)
 
         return result
+
+    @classmethod
+    def commit(cls, occ, external_unique_number, options = None):
+
+        path = cls.__TRANSACTION_BASE_PATH + cls.__COMMIT_TRANSACTION
+        api_base = onepay.integration_type.value.api_base
+
+        req = TransactionCommitRequest(occ, external_unique_number, int(datetime.now().timestamp()), options)
+
+        try:
+            data_response = requests.post(api_base + path, data = TransactionCommitRequestSchema().dumps(req).data).text
+        except Exception:
+            raise TransactionCreateError("Could not obtain a response from the service")
```

Comment:
Are we losing the original exception information? Sounds like even your future self might hate this decision when he has to debug a case where this exception was raised